Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG BASE_IMAGE=golang:1.22-bookworm
```

Comment:
go1.22.x is already in CI?

Reply:
For actions/setup-go, "1.21" and "1.21.x" should be equivalent, as shown [here](https://github.com/goplus/gop/actions/runs/8223774578/job/22486829010#step:8:11). Therefore, merging this PR should not pose any issues.

However, I'll still update `.github/workflows/release-build.yml` to use "1.21.x" to align with `.github/workflows/go.yml`. This will be done in a separate PR along with other formatting changes.